packages/block-editor/src/components/link-control/settings-drawer.js

```diff
@@ -46,7 +46,7 @@ function LinkSettingsDrawer( {
 				aria-expanded={ settingsOpen }
 				onClick={ () => setSettingsOpen( ! settingsOpen ) }
 				icon={ settingsIcon }
-				label={ __( 'Toggle link settings' ) }
+				label={ __( 'Link settings' ) }
```

Comment:
There have been numerous discussions about the use of the word "Toggle" in WordPress. It should be avoided because it cannot be translated in other languages. In this case, it was superfluous.

Reply:
...agreed. We removed that word in a recent PR 👍